pennylane/devices/qubit/simulate.py

```diff
@@ -65,6 +65,40 @@
 }
 
 
+class TreeTraversalStack:
+    """This class is used to record various data used during the
+    depth-first tree-traversal procedure for simulating dynamic circuits."""
+
+    counts: Sequence
```

Comment:
Would this be allowed to be a tuple? Do we allow it to be any type of sequence, or specifically a mutabe sequence (list)?

Reply:
They should be lists actually since we use item assignment.